File: MetalSampleCodeLibrary/RenderWorkflows/LoadingTexturesAndModelsUsingMetalFastResourceLoading/AAPLAppDelegate.m
/*
See LICENSE folder for this sample’s licensing information.

Abstract:
The implementation of the iOS app delegate.
*/
#import "AAPLAppDelegate.h"

@interface AAPLAppDelegate ()
@end

@implementation AAPLAppDelegate

- (void) applicationDidFinishLaunching:(NSNotification*) aNotification
{
    // Insert code here to initialize your app.
}

- (void) applicationWillTerminate:(NSNotification*) aNotification
{
    // Insert code here to tear down your app.
}

- (BOOL) applicationShouldTerminateAfterLastWindowClosed:(NSApplication*) sender
{
    return YES;
}

@end

File: MetalFX/ApplyingTemporalAntialiasingAndUpscalingUsingMetalFX/Application/macOS/AAPLAppDelegate.swift
/*
See the LICENSE.txt file for this sample’s licensing information.

Abstract:
This file provides the app delegate for the MetalFX sample.
*/

import Cocoa

@main
class AAPLAppDelegate: NSObject, NSApplicationDelegate {
    @IBOutlet var window: NSWindow!

    func applicationDidFinishLaunching(_ aNotification: Notification) {
    }

    func applicationWillTerminate(_ aNotification: Notification) {
    }

    func applicationShouldTerminateAfterLastWindowClosed(_ sender: NSApplication) -> Bool {
        return true
    }
}
